src/mpl/src/gavl/mpl_gavl.c

```diff
@@ -550,7 +594,33 @@ int MPL_gavl_tree_delete(MPL_gavl_tree_t gavl_tree, const void *addr, uintptr_t
     };
 
   fn_exit:
-    return mpl_err;
-  fn_fail:
-    goto fn_exit;
+    return;
+}
+
+/* gavl_tree_delete_removed_nodes deletes all nodes intersects input key (addr, len) in remove_list */
+static void gavl_tree_delete_removed_nodes(gavl_tree_s * tree_ptr, uintptr_t addr, uintptr_t len)
```

Comment:
Grammar fix: "deletes all nodes intersects input key..." -> "delete all nodes that intersect the input key..."